apps/kaizen/models.py

```
@@ -28,30 +24,35 @@ class SubTask(BaseModel):
+        (4, "Productive"),
+        (5, "Very Productive"),
     ]
 
-    date = models.DateField()
     title = models.CharField(max_length=255)
-    reflection = models.TextField()
-    rating = models.IntegerField(choices=RATING_CHOICES)
+    notes = models.TextField(blank=True, default="", null=True)
+    mood = models.IntegerField(choices=MOOD_CHOICES, default=None)
+    productivity_rating = models.IntegerField(
+        choices=PRODUCTIVITY_CHOICES, default=None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)